webrtc.html

```diff
@@ -2251,6 +2251,10 @@
             state. Note the local or remote SDP descriptions in the
 	    previous stable state could be null if there has
             not yet been a successful offer-answer negotiation.</p>
+
+            <p>If the value of a <code>RTCRtpTransceiver.mid</code> was set to a non-null value by
+              an SDP description that is rolled back, the <code>RTCRtpTransceiver.mid</code> will be
+              set to null, as defined by <span data-jsep="rollback">[[!JSEP]]</span>.</p>
```

Comment:
Isn't this a bit ambiguous in the sense that if an SDP sets the mid to one value (x), and a subsequent SDP sets it to another value (y), at rollback of the latest SDP I assume the mid should roll back to x and not to null?